package.json

```diff
@@ -20,7 +20,9 @@
     "eslint": "^7.27.0",
     "fs-extra": "^9.0.1",
     "glob": "^7.1.6",
+    "handlebars": "^4.7.7",
     "html-entities": "^2.3.2",
+    "moment": "^2.29.1",
```

Comment:
By the way is there a reason why all the dependencies are under devDependencies? Doesn't it mean that the libs won't be packaged with the plugin when you publish?

Reply:
Won't webpack automatically include it in the bundled js?

Reply:
I don't know what webpack does, but devDependencies as the name says is for development - so for example the TypeScript package, eslint, etc. However packages you actually used in the plugin such as  handlebars or moment should be under the "dependencies" key.

Reply:
Oh ok, I'll create a PR for correcting this.